app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user && user.authenticate(params[:session][:password])
```

Comment:
Tìm hiểu toán tử `&.` có tác dụng gì em nhé
```suggestion
    if user&.user.authenticate(params[:session][:password])
```

Reply:
Toán tử &. có tác dụng là kiểm tra xem đối tượng có phải là nil hay không. Nếu là nil thì nó sẽ trả về nil còn không thì nó sẽ gọi đến phương thức mà đối tượng đã gọi sau toán tử.